samples/ipc_call_interface/producers/cipher_v2i1/interfaces/cipher_v2i1_interface.h

```diff
@@ -18,6 +18,9 @@ extern "C"
 #else
 #endif
 
+#define CIPHER_2_PACKAGE_NAME "cipher"
+#define CIPHER_2_PACKAGE_VERSION 2
```

Comment:
Is this the final way people will set their package names and versions? You could stringify the file #define and then split the string for the package name and version

Reply:
I don't think we should do that, it will add code that is not necessary.